dist/api_data.py

```diff
@@ -505,13 +505,18 @@ def __ge__(self, other):
             return a WT_ROLLBACK error from a transaction operation about
             every Nth operation to simulate a collision''',
             min='0', max='10M'),
+        Config('slow_checkpoint', 'false', r'''
+            if true, slow down checkpoint creation by slowing down internal
+            page processing.
+            ''',
+            type='boolean'),
         Config('table_logging', 'false', r'''
             if true, write transaction related information to the log for all
             operations, even operations for tables with logging turned off.
             This setting introduces a log format change that may break older
             versions of WiredTiger. These operations are informational and
             skipped in recovery.''',
-            type='boolean'),
+            type='boolean')
```

Comment:
All of the categories have the final entries with trailing comma. I suggest leaving this alone.